streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollector.java

```diff
@@ -81,7 +81,9 @@ public RecordCollector(Producer<byte[], byte[]> producer) {
             if (partitions != null)
                 partition = partitioner.partition(record.key(), record.value(), partitions.size());
         }
-        this.producer.send(new ProducerRecord<>(record.topic(), partition, keyBytes, valBytes), callback);
+        this.producer.send(new ProducerRecord<>(record.topic(), partition, record.timestamp(),
```

Comment:
`StoreChangeLogger` doesn't have the timestamp info. It just has the Key and Value i believe.
Can we look into this in another JIRA/PR?